apps/site/package.json

```
@@ -61,7 +61,7 @@
     "@vercel/otel": "~1.10.0",
     "@vercel/speed-insights": "~1.1.0",
     "autoprefixer": "~10.4.20",
-    "classnames": "~2.5.1",
+    "clsx": "^2.1.1",
```

```suggestion
    "clsx": "~2.1.1",
```

IMO we should keep `~`; https://github.com/nodejs/nodejs.org/blob/main/DEPENDENCY_PINNING.md?plain=1#L23